gatsby-node.js

```diff
@@ -368,4 +373,16 @@ exports.createPages = async ({ reporter, actions, graphql }) => {
             },
         })
     })
+    const videos = result.data.directus.videos
```

Comment:
We already had the same logic for blog in L364. Since we had no issue with that I think we wont get any error in future. But we can use optional chaining if you approve we need.